content/_partials/changelog-old.html

```diff
@@ -6993,7 +6993,7 @@ <h3 id=v1.388>What's new in 1.388 (2010/12/04)</h3>
     Fixed an <tt>AbstractMethodError</tt> in listing up executors.
     (<a href="https://issues.jenkins.io/browse/JENKINS-8106">issue 8106</a>)
   <li class=bug>
-    Slaves launched by JNLP fail to reprot their version numbers.
+    Agents launched by JNLP fail to reprot their version numbers.
```

Comment:
Since we're changing the line, should we correct the spelling also?
```suggestion
    Agents launched by JNLP fail to report their version numbers.
```